include/boost/gil/color_convert.hpp

```diff
@@ -150,44 +150,46 @@ struct default_color_converter_impl<rgb_t,gray_t> {
 /// by Burger, Wilhelm, Burge, Mark J.
 /// and it is a gross approximation not precise enough for professional work.
 ///
-/// \todo FIXME: The original implementation did not handle properly signed CMYK pixels as destination
-///
 template <>
 struct default_color_converter_impl<rgb_t, cmyk_t>
 {
     template <typename SrcPixel, typename DstPixel>
     void operator()(SrcPixel const& src, DstPixel& dst) const
     {
         using src_t = typename channel_type<SrcPixel>::type;
-        src_t const r = get_color(src, red_t());  
-        src_t const g = get_color(src, green_t());
-        src_t const b = get_color(src, blue_t());
+        src_t const r  = get_color(src,red_t());
+        src_t const g  = get_color(src,green_t());
+        src_t const b  = get_color(src,blue_t());
```

Comment:
Please, avoid any stylistic or formatting changes in the existing code as in [DO NOT mix independent, unrelated changes in one PR](https://github.com/boostorg/gil/blob/develop/CONTRIBUTING.md#pull-requests)

Reply:
Ok I'll work on adding tests.
Also I would be thankful if you provide me some link for doing the same.

Reply:
As a first stab at tests, you can just add it as a new program, e.g. `gil/test/core/color/default_color_converter_rgb_to_cmyk.cpp`, based on my sample in #479.

Eventually, we may aggregate it into this [test/core/color/default_color_converter_impl.cpp](https://github.com/boostorg/gil/blob/01d95998c278d113af32ed34ba7d1ccc5987e4be/test/core/color/default_color_converter_impl.cpp).

By the way, the `test/core/color/default_color_converter_impl.cpp` is minimal and it would be good to cover wider range(s) of test values.

Reply:
Ok, thanks.

Reply:
My note on  `test/core/color/default_color_converter_impl.cpp` being minimal is just in case you wondered, why it does not cover wider range of values. I'd be happy if it did -  there is still plenty of work to do in GIL tests (help is highly welcome!).

So, I encourage you to consider writing better tests for this fix than what we've got in `test/core/color/default_color_converter_impl.cpp` so far :)

Reply:
That would be exciting to do, thanks for that.
I would get to learn new things.

Reply:
👍 Warning: it's not easy, it's sometimes boring or frustrating. I'll try to help :)

Reply:
Ok, thanks.